pkg/sql/exec/operator.go

```diff
@@ -47,6 +47,14 @@ type OpNode interface {
 	Child(nth int) OpNode
 }
 
+// NonExplainable is a marker interface which identifies an Operator that
+// should be omitted from the output of EXPLAIN (VEC). Note that VERBOSE
+// explain option will override the omitting behavior.
+type NonExplainable interface {
+	// marker is just a marker method. It should never be called.
+	marker()
```

Comment:
I think you should make this something like `nonExplainableMarker` instead to avoid namespace clashes, since duck typing is done by method name. Not that this will necessarily happen, but better to be careful if we can.